src/test/java/com/openshift/client/fakes/UserConfigurationFake.java

```diff
@@ -53,7 +53,7 @@ protected void initFile(File file) {
 			try {
 				StreamUtils.close(writer);
 			} catch (Exception e) {
-				throw new RuntimeException(e);
```

Comment:
-1
This is done on purpose. Throwing runtime exceptions vs checked exceptions is a design choice. You can read more about it for instance http://www.javapractices.com/topic/TopicAction.do?Id=129 and in other places.